src/routes/UserRoute.tsx

```diff
@@ -10,24 +11,21 @@ import {
   USER_PREFERENCES_ROUTE,
 } from './routeConstants';
 
-const navigationItems = [
+const navigationItems: SelectOption[] = [
   {
-    id: 0,
-    linkTo: USER_EDIT_PROFILE_ROUTE,
+    to: USER_EDIT_PROFILE_ROUTE,
     label: 'Profile',
-    value: 'profile',
+    value: 0,
```

Comment:
What do these values mean?

Wound't a `enum` make them descriptive ?

Reply:
This nav will be removed soon. because of a new design. I think that we can skip it right now. Unless there's a reason to spend time on it right now.